src/celeritas/io/ImportMaterial.hh

```diff
@@ -59,12 +59,11 @@ struct ImportMatElemComponent
  */
 struct ImportMaterial
 {
-    //// TYPES ////
-
+    //!@{
+    //! Type aliases
```

Comment:
I think either representation is fine for `struct`s ... originally my idea was that structs should be so simple you don't need much more than dividers (`//// TYPES ////`) but the doxygen descriptors are good too. By the way, the documentation rendering works better if this is a "group" using `//! \name Type aliases` rather than copying the description `Type aliases` to each one of these. I updated the template a while back but didn't explain why.